internal/cmd/operator-sdk/pkgmantobundle/cmd.go

```diff
@@ -0,0 +1,311 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkgmantobundle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	apimanifests "github.com/operator-framework/api/pkg/manifests"
+	"github.com/operator-framework/operator-sdk/internal/annotations/metrics"
+	"github.com/operator-framework/operator-sdk/internal/util/bundleutil"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	longHelp = `
+'pkgman-to-bundle' command helps in migrating OLM packagemanifests to bundles which is the preferred OLM packaging format.
+This command takes an input packagemanifest directory and generates bundles for each of the versions of manifests present in
+the input directory. Additionally, it also provides the flexibility to build bundle images for each of the generated bundles.
+
+The generated bundles are always written on disk. Location for the generated bundles can be specified using '--output-dir'. If not
+specified, the default location would be 'bundle/' directory.
+
+The base container image name for the bundles can be provided using '--base-image' flag. This should be provided without the tag, since the tag
```

Comment:
```suggestion
The base container image name for the bundles can be provided using '--image-tag-base' flag. This should be provided without the tag, since the tag
```